src/pages/index.js

```diff
@@ -44,39 +55,115 @@ const addImageCardCloseButton =
   addImageCardModal.querySelector(".modal__close");
 const imageCaption = addImageCardModal.querySelector(".modal__caption");
 const modalImage = addImageCardModal.querySelector(".modal__image");
-function handleEscape(evt) {
-  if (evt.key === "Escape") {
-    const openModal = document.querySelector(".modal_opened");
-    if (openModal) {
-      closePopup(openModal);
-    }
-  }
-}
+
+// function handleEscape(evt) {
+//   if (evt.key === "Escape") {
+//     const openModal = document.querySelector(".modal_opened");
+//     if (openModal) {
+//       closePopup(openModal);
+//     }
+//   }
+// }
 
 function handleCardClick(data) {
   popupWithImage.open(data);
 }
 
 function handleProfileEditSubmit(inputValues) {
-  userInfo.setUserInfo(inputValues.title, inputValues.description);
-  popupEditWithForm.close();
+  popupEditWithForm.renderLoading(true);
+  api
+    .editProfileInfo(inputValues)
+    .then((data) => {
+      userInfo.setUserInfo(data.name, data.about);
+      popupEditWithForm.close();
+      popupEditWithForm.renderLoading(false);
```

Comment:
You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case: `success` or `failure`.

**Please, correct it everywhere in the code**